tt_metal/hostdevcommon/profiler_common.h

```diff
@@ -24,18 +24,16 @@ namespace kernel_profiler{
 
     enum ControlBuffer
     {
-        HOST_BUFFER_END_INDEX_BR,
+        HOST_BUFFER_END_INDEX_BR_ER,
```

Comment:
is this because eth cores treated as if there is one brisc? this may not be scalable for BH which has 2 riscvs on eth cores

Reply:
Good call, this will turn into a union and each device can define its own struct for controls.